ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -145,5 +114,20 @@ public async Task<IList<Notification>> GetNotificationsByIdsAsync(IList<int> ids
                         .Where(n => ids.Contains(n.NotificationId))
                         .ToListAsync();
         }
+
+        private IQueryable<Notification> GetBaseNotificationIQueryable()
+        {
+            return context.Notification
+                .Include(n => n.PatientDetails)
+                    .ThenInclude(p => p.Sex)
```

Comment:
ah I didn't realise you only needed ThenInclude for collections and could chain them otherwise - not sure whether it's worth changing, but if so should probably be consistent and do it everywhere? What do you think @jasiekmiko ?

Reply:
Yeah I think we'd still need separate lines like this - let's just leave it for now!